class/class-mainwp-child-updraft-plus-backups.php

```
@@ -2246,10 +2246,8 @@ private function analyse_db_file_old( $timestamp, $res, $db_file = false, $heade
          */
         global $updraftplus;
 
-        $wp_ver = MainWP_Child_Server_Information_Base::get_wordpress_version();
-
-
-        include_once ABSPATH . WPINC . '/version.php'; // NOSONAR - WP compatible.
+        // Get WordPress version using MainWP_Child_Server_Information_Base.
+        $wp_version = \MainWP\Child\MainWP_Child_Server_Information_Base::get_wordpress_version();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$wp_version'. (Unused Code Rules)

(UnusedLocalVariable)